pages/implementation/start-custom-destination.mdx

```diff
@@ -381,6 +381,31 @@ import Avo
 {
     // TODO implement
 }
+
+- (void)logPage:(nonnull NSString *)pageName withEventProperties:(nonnull NSDictionary*)eventProperties
+{
+    // TODO implement
+}
+
+- (void)revenue:(double)amount withEventProperties:(nonnull NSDictionary*)eventProperties
+{
+    // TODO implement
+}
+
+- (void)setUserProperties:(nonnull NSString*)userId withUserProperties:(nonnull NSDictionary*)userProperties
+{
+    // TODO implement
+}
+
+- (void)identify:(nonnull NSString*)userId
+{
+    // TODO implement
+}
+
+- (void)unidentify
+{
+    // TODO implement
```

Comment:
Opportunity here to improve the `// TODO implement`, to make it look less like an internal todo on our end. Maybe something like `// TODO: implement an identify call to your analytics SDK here`